src/pages/contact.js

```diff
@@ -20,6 +21,7 @@ const ContactPage = () => {
       <h1>Contact Info</h1>
       <p>{contactInfo.contact.email}</p>
       <a href={contactInfo.contact.links.twitter}>twitter</a>
+      <a href={contactInfo.contact.links.facebook}>facebook</a>
```

Comment:
@a1cha Could you wrap these both in paragraph tags (`<p>`) to give space between the two links? An anchor tag (`<a>`) is an inline element so it will show on the same line right beside the next element if the next element doesn't create a line break

![Screen Shot 2020-10-28 at 4 32 58 PM](https://user-images.githubusercontent.com/21016245/97495984-b3320a80-193e-11eb-8995-0653120c89a3.png)

```suggestion
      <p><a href={contactInfo.contact.links.twitter}>twitter</a></p>
      <p><a href={contactInfo.contact.links.facebook}>facebook</a></p>
```

Reply:
@alexxmitchell done. Sorry for the mess.